src/WorkflowCore/Models/WorkflowOptions.cs

```diff
@@ -34,6 +34,11 @@ public WorkflowOptions(IServiceCollection services)
             EventHubFactory = new Func<IServiceProvider, ILifeCycleEventHub>(sp => new SingleNodeEventHub(sp.GetService<ILoggerFactory>()));
         }
 
+        public bool DisableWorkflowConsumer { get; set; } = false;
+        public bool DisableEventConsumer { get; set; } = false;
+        public bool DisableIndexConsumer { get; set; } = false;
+        public bool DisableRunnablePoller { get; set; } = false;
```

Comment:
What do you think of flipping the naming here and default to true... something like `ProcessWorkflows`, `ProcessEvents`, `PollingEnabled` or some variant?

Reply:
Yes, no problem